### PR TITLE
Added Clear Error and Warn Messages when API is unavailable

### LINKED ISSUE #535: Clear Messaging when the Database is Offline
**Describe the bug**
We need clear messaging that the API is offline when any of these events happen:

1. The Talawa-Admin app is started
    1. See this issue as a reference: https://github.com/PalisadoesFoundation/talawa-api/issues/1122
1. As an Admin attempts to login
1. As an Admin is using the various dashboards

This will help both the end user and developers. This may need to be split into three separate issues.

**To Reproduce**

- See above

**Expected behavior**

- See above

**Actual behavior**

- See above

**Screenshots**

N/A

**Additional details**

- See above

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by aviraldevv**
**What kind of change does this PR introduce?**
This PR adds clear error and warn messages when Talawa-API is unavailable. 
It solves two out of three issues mentioned in its parent isssue #535 

**Issue Number:**
  #535 

**Did you add tests for your changes?**
Yes

**Snapshots/Videos:**
Screenshot_412

Screenshot_413

**If relevant, did you update the documentation?**
Not Required

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

**Comment by palisadoes** [comment]
@aviraldevv 

- The success message is from the coder perspective. The user won't know what that means. Think of something else.
- If there is no API access, then the message should be red.

**Comment by aviraldevv** [comment]
@palisadoes How about `Talawa-Admin is up and running !` ??

**Comment by palisadoes** [comment]
That's OK

**Comment by palisadoes** [comment]
@noman2002 @Kevoniat Please review

**Comment by aviraldevv** [comment]
@palisadoes I have done the required changes .

**Comment by palisadoes** [comment]
@noman2002 @Kevoniat please review

**Review by noman2002 — APPROVED** [review]

**Comment by aviraldevv** [comment]
@palisadoes @noman2002 please don't merge it for now, I have just noticed that when user logs out, it is showing the success message which not a good user experience to be honest. I'll fix it up and update. I am really sorry for the inconvenience.

**Comment by aviraldevv** [comment]
@palisadoes @noman2002 please don't merge this for now. I have just noticed that it is showing success message when the user logs out which was not expected. I will fix the code. I am really sorry for the inconvenience.

**Comment by palisadoes** [comment]
1. Is it ready now?
2. Do you have a test for this new occurrence? It is and important test to have.

**Comment by aviraldevv** [comment]
@palisadoes I will push my final changes after last cross check right now will inform you when its ready to merge. Also will submit the PR for rest of the screens today only.

**Comment by aviraldevv** [comment]
@palisadoes I am done with my changes we can merge this now. No extra changes have been made since @noman2002 review, I have only omitted the success functionality.

**Review by palisadoes — APPROVED** [review]

File: src/screens/LoginPage/LoginPage.tsx [hunk 1]
```diff
@@ -70,6 +70,27 @@ function LoginPage(): JSX.Element {
   const [recaptcha, { loading: recaptchaLoading }] =
     useMutation(RECAPTCHA_MUTATION);
 
+  useEffect(() => {
+    async function loadResource() {
+      const resourceUrl = 'http://localhost:4000/graphql/';
```
[the remark this answers is not in the document]
Comment (aviraldevv) [reply src/screens/LoginPage/LoginPage.tsx:75]: @anshgoyalevil Sure I will change it my next PR, thanks for the input.

**Pull request merged**